paper-ieee-trans.tex

```diff
@@ -261,14 +261,14 @@ \subsection{Enabling Grids for E--sciencE}
 official software distribution of {\sl EGEE} as of 2006, after two years of prototyping and
 re--engineering efforts to converge with the {\sl LHC} Computing Grid (LCG--2), Virtual
 Data Toolkit (VDT) and Condor \cite{condor} software stacks. The
-development team was comprised of more than 80 people from 12 academic and
-industrial partners, which issued more than 10 thousand bug, 1.7 thousand patches and
-defined over 300 development tasks tracked by the use of bug/task management tools.
+development team was comprised of more that 80 people from 12 academic and
```

Comment:
than

Reply:
agree with change
make "that" -> "than"

Reply:
ok, sorry I didn't catch the meaning :)

Reply:
yes, more than....

Reply:
ok. Sorry probably i've read to fast the phrase.